src/main/kotlin/com/example/mykku/board/dto/UpdateBoardRequest.kt

```
@@ -0,0 +1,9 @@
+package com.example.mykku.board.dto
+
+data class UpdateBoardRequest(
+    val id: Long,
+    val title: String,
+    val logo: String
+) {
+
+}
```

detekt (1.23.8)

[warning] The class or object UpdateBoardRequest is empty.

(detekt.empty-blocks.EmptyClassBlock)